README.md

```diff
@@ -69,10 +72,11 @@ Alternatively, you can organize your login information with file structure. For
   * mysite.com
 
 PassFF will
+* get the user from the "user" file unser supersite.com
 * get the login from the "login" file under supersite.com
 * get the login from the "login" field inside the mysite.com entry for mysite.com (see [format above](#multi-line-format))
 
-The file structure approach does not support custom input fields, however.
+The file structure approach does not support custom input fields, however. Also, you cannot use this layout with [the format above](#multi-line-format) for cases where your username is `user` (which has been encountered in the wild in some old devices), or another reserved name.
```

Comment:
```suggestion
The file structure approach does not support custom input fields.

Note that the file structure format is recognized and assumed by PassFF whenever a file name matches a reserved field name such as `user`, `url`, `password` or `login`. This might cause unexpected behavior in cases where there is a file in [multi-line format](#multi-line-format) whose name happens to be a reserved field name.
```